packages/react/__tests__/src/components/Pagination/index.js

```diff
@@ -28,29 +28,39 @@ describe('Pagination', () => {
           wrapper
             .find('button')
             .at(0)
-            .is('[aria-disabled="true"]')
-        ).toBe(true);
+            .getDOMNode()
+            .getAttribute('aria-disabled')
+        ).toBe('true');
 
         expect(
           wrapper
             .find('button')
             .at(1)
-            .is('[aria-disabled="true"]')
-        ).toBe(true);
+            .getDOMNode()
+            .getAttribute('aria-disabled')
+        ).toBe('true');
 
+        console.log(
```

Comment:
👀

Reply:
Good 👀 . I'll fix it. At least it was just a test file

Reply:
https://github.com/dequelabs/cauldron/pull/941